build.gradle

```diff
@@ -266,6 +164,113 @@ subprojects {
   }
 }
 
+def pomFiller(pom, nameVar, descriptionVar) {
+  pom.project {
+    name = nameVar
+    description = descriptionVar
+    url = 'http://ehcache.org'
+    organization {
+      name = 'Terracotta Inc., a wholly-owned subsidiary of Software AG USA, Inc.'
+      url = 'http://terracotta.org'
+    }
+    issueManagement {
+      system = 'Github'
+      url = 'https://github.com/ehcache/ehcache3/issues'
+    }
+    scm {
+      url = 'https://github.com/ehcache/ehcache3'
+      connection = 'scm:git:https://github.com/ehcache/ehcache3.git'
+      developerConnection = 'scm:git:git@github.com:ehcache/ehcache3.git'
+    }
+    licenses {
+      license {
+        name = 'The Apache Software License, Version 2.0'
+        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+        distribution = 'repo'
+      }
+    }
+    developers {
```

Comment:
Need to get an answer for that and create an issue to clean up / complete for milestone 4